blocks/hero-home/hero-home.css

```
@@ -0,0 +1,78 @@
+video {
```

```suggestion
video {
```
it would be better to improve specificity with .hero-home video, to reduce risk of multiple video css definitions in other blocks